webui/react/src/components/kit/Breadcrumb.tsx

```diff
@@ -22,4 +24,8 @@ const Breadcrumb: Breadcrumb = AntdBreadcrumb;
 Breadcrumb.Item = AntdBreadcrumb.Item;
 Breadcrumb.Separator = AntdBreadcrumb.Separator;
 
+export const BreadcrumbBar: React.FC<React.PropsWithChildren> = ({ children }) => {
+  return <div className={css.base}>{children}</div>;
```

Comment:
Why do we need extra styles here? Can we just use the BreadCrumb component as-is?

Reply:
I'm not attached to the bar and the breadcrumb component in the UI Kit was signed off by design at the beginning of the project.